core/common/src/main/java/alluxio/wire/MountPointInfo.java

```diff
@@ -0,0 +1,186 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.wire;
+
+import alluxio.Configuration;
+import alluxio.underfs.UnderFileSystem;
```

Comment:
We want the common module to depend on underfs.

The wire type should simply be a set of fields with a getter and setter for each. Any non-trivial logic should be done in the caller.